apps/docs/components/Utils/ThemeProvider/ThemeSwitch/themeSwitch.css

```diff
@@ -0,0 +1,16 @@
+.hd-themeSwitchButton {
```

Comment:
Why did you change the way you name the classes? Based on the BEM methodology. 

Example:
`hd-theme-switch__button` or `hd-theme-switch__btn`